tests/manual-test-cases/Group18-VIC-UI/18-3-VIC-UI-NGC-tests.robot

```diff
@@ -56,6 +63,16 @@ Run Ngc Tests Project
 
 
 *** Keywords ***
+Update Config To Use Webserver For Plugin Testing
```

Comment:
`vicui-common.robot` is pretty much the place for those common stuff. I'll try refactoring some existing code.